Xcode/SDL/pkg-support/build.xcconfig

```diff
@@ -0,0 +1,8 @@
+//
+//  build.xcconfig
+//
+
+// Configuration settings file format documentation can be found at:
+// https://help.apple.com/xcode/#/dev745c5c974
+
+SDL3_PREPROCESSOR_DEFINITIONS = SDL_VENDOR_INFO="libsdl.org"
```

Comment:
```suggestion
SDL_PREPROCESSOR_DEFINITIONS = SDL_VENDOR_INFO=\"libsdl.org\"
```
I've added config.xcconfig separately, so you can remove that from this PR, and this is the line you want here.

Reply:
Thanks!